### PR TITLE
chore(workflow-v2.3.0): Release workflow and workflow-e2e version 2.3.0

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: router-v2.3.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: router-dev
+home: https://github.com/deis/router
+version: v2.0.0
```
Comment (slack) [inline router-v2.3.0/Chart.yaml]: This should be `v2.3.0`?

File: router-v2.3.0/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: router-dev
```
Comment (slack) [inline router-v2.3.0/Chart.yaml]: This should be the release version as well.

File: workflow-v2.3.0-e2e/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: workflow-dev-e2e
+home: https://github.com/deis/workflow-e2e
+version: v2.0.0
```
Comment (slack) [inline workflow-v2.3.0-e2e/Chart.yaml]: `version` and `name` need to be updated.

File: workflow-v2.3.0-e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Workflow 2.0.0-dev e2e Tests (parallel)
```
Comment (slack) [inline workflow-v2.3.0-e2e/README.md]: 2.3.0

File: workflow-v2.3.0/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Workflow dev
+
+WARNING: this chart is for testing only! Features may not work, and there are likely to be bugs.
```
Comment (slack) [inline workflow-v2.3.0/README.md]: Warning should be removed.

File: workflow-v2.3.0/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+name: workflow-dev
```
Comment (slack) [inline workflow-v2.3.0/Chart.yaml]: Name and version need to be fixed here as well.

File: workflow-v2.3.0-e2e/tpl/generate_params.toml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+[e2e]
+org = "deisci"
+dockerTag = "v2.3.0"
+pullPolicy = "Always"
+podName = "workflow-dev-e2e"
```
Comment (vdice) [inline workflow-v2.3.0-e2e/tpl/generate_params.toml]: This should be `workflow-v2.3.0-e2e`

File: workflow-v2.3.0-e2e/tpl/generate_params.toml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+[e2e]
+org = "deisci"
```
Comment (vdice) [inline workflow-v2.3.0-e2e/tpl/generate_params.toml]: @jchauncey the org here should be `"deis"`

File: router-v2.3.0/Chart.yaml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+name: router-v2.3.0
+home: https://github.com/deis/router
+version: v2.3.0
+description: Deis Router (For testing only!)
```
Comment (slack) [inline router-v2.3.0/Chart.yaml]: Drop "For testing only"!

Comment (slack) [inline router-v2.3.0/Chart.yaml]: Nit, btw, not a blocker.

Comment (kmala) [inline router-v2.3.0/Chart.yaml]: updated.

**Pull request merged**